cups/md5passwd.c

```diff
@@ -28,22 +30,7 @@ httpMD5(const char *username,		/* I - User name */
         const char *passwd,		/* I - Password string */
 	char       md5[33])		/* O - MD5 string */
 {
-  unsigned char		sum[16];	/* Sum data */
-  char			line[256];	/* Line to sum */
-
-
- /*
-  * Compute the MD5 sum of the user name, group name, and password.
-  */
-
-  snprintf(line, sizeof(line), "%s:%s:%s", username, realm, passwd);
-  cupsHashData("md5", (unsigned char *)line, strlen(line), sum, sizeof(sum));
-
- /*
-  * Return the sum...
-  */
-
-  return ((char *)cupsHashString(sum, sizeof(sum), md5, 33));
+  return NULL;
```

Comment:
OK, code style and stubbing issues here. The original function put a nul-terminated string in the "md5" buffer, so we need to do that (`md5[0] = '\0';`) and then put out return value in parenthesis (a style I adopted decades ago for a bunch of reasons...)